nouwatinjacob/Politico
# #164105124 Setup database

#### What does this PR do?
This PR set up the application database and setup the Party model
#### Description of Task to be completed?
- Add the postgres database setting
- create party model
#### How should this be manually tested?
- git clone this repo by `git clone https://github.com/nouwatinjacob/Politico.git`
- checkout to the branch with `git checkout ch-setup-database-164105124`
- run `python manage.py makemigrations`
- run `python manage.py migrate`
- You if you check your database the party table m ust have been created with other auth tables
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[164105124](https://www.pivotaltracker.com/story/show/164105124)
#### Screenshots or gifs (if appropriate)
N/A
#### Questions:
N/A
